app/components/LaptopMockup.tsx

```diff
@@ -0,0 +1,32 @@
+import { FC } from "react";
```

Comment:
If you remove this, does it still give you an error? You are restructuring your props which should implicitly inform Typescript of the types.